packages/modules/vehicles/mercedeseq/config.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional
+
+
+class MercedesEQSocToken:
+    def __init__(self,
+                 refresh_token: str = "",
+                 access_token: str = "",
+                 expires_in: int = 0,
+                 id_token: str = "",
+                 token_type: str = "") -> None:
+        self.refresh_token = refresh_token
+        self.access_token = access_token
+        self.expires_in = expires_in
+        self.id_token = id_token
+        self.token_type = token_type
+
+
+class MercedesEQSocConfiguration:
+    def __init__(self, client_id: Optional[str] = None, client_secret: Optional[str] = None, vin: Optional[str] = None,
+                 callbackurl: Optional[str] = None, token: MercedesEQSocToken = None) -> None:
```

Comment:
```suggestion
                 token: MercedesEQSocToken = None) -> None:
```

Reply:
Wir können die callbackurl nicht raus nehmen, in der auth.py wird die callbackurl benötigt um die Token abzufragen. 
```
callback = conf['configuration']['callbackurl']
printDebug("ClientID:"+client_id[0:3] + "**********" + client_id[-3:0], 10)
printDebug("ClientSecret: " + client_secret[0:3] + "**********" + client_secret[-3:], 10)
printDebug("Callback:" + callback, 10)

tok_url = "https://ssoalpha.dvb.corpinter.net/v1/token"

data = {'grant_type': 'authorization_code', 'code': str(code), 'redirect_uri': callback}
```

Reply:
@benderl : Vermute, du musst die Callbackurl dann füllen, damit ich sie wieder auslesen kann.